tests/test_tacacs_global.rb

```diff
@@ -0,0 +1,79 @@
+#
+# Minitest for TacacsGlobal class
+#
+# Copyright (c) 2014-2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/tacacs_global'
+
+# TestTacacsGlobal - Minitest for TacacsGlobal node utility.
+class TestTacacsGlobal < CiscoTestCase
+  @skip_unless_supported = 'tacacs_global'
+
+  def setup
+    # setup runs at the beginning of each test
+    super
+    config_no_warn('feature tacacs+') if platform == :nexus
+    no_tacacs_global
+  end
+
+  def teardown
+    # teardown runs at the end of each test
+    no_tacacs_global
+    super
+  end
+
+  def no_tacacs_global
+    # Turn the feature off for a clean test.
+    config('no tacacs-server timeout 2')
```

Comment:
This doesn't turn off the tacacs feature. Is this a sufficient cleanup? will it cleanup stale keys from a previous test?